webapp/javascript/ui/Dropdown.tsx

```diff
@@ -48,6 +51,15 @@ export default function Dropdown({
   menuButtonClassName = '',
   ...props
 }: DropdownProps) {
+  const menuButtonComponent = props.menuButton || (
+    <MenuButton
+      className={`${styles.dropdownMenuButton} ${menuButtonClassName}`}
```

Comment:
Maybe we can use https://www.npmjs.com/package/classnames here too?

Reply:
done